document_merge_service/api/engines.py

```diff
@@ -1,5 +1,6 @@
 import io
 import zipfile
+from collections import UserString
```

Comment:
> This UserString class from this module is available for backward compatibility only. If you are writing code that does not need to work with versions of Python earlier than Python 2.2, please consider subclassing directly from the built-in str type instead of using UserString

https://docs.python.org/2.4/lib/module-UserString.html

Reply:
:+1: